src/monorepo-workflow-operations.test.ts

```diff
@@ -1,1788 +1,1214 @@
-                      a: new SemVer('1.0.0'),
-                    },
-                  },
-                });
-
-                await expect(
-                  followMonorepoWorkflow({
-                    project,
-                    tempDirectoryPath: sandbox.directoryPath,
-                    firstRemovingExistingReleaseSpecification: false,
-                    stdout,
-                    stderr,
-                  }),
-                ).rejects.toThrow(expect.anything());
-
-                expect(
-                  await fs.promises.stat(
-                    path.join(sandbox.directoryPath, 'RELEASE_SPEC'),
-                  ),
-                ).toStrictEqual(expect.anything());
-              });
+      it('removes the release spec file at the end', async () => {
+        await withSandbox(async (sandbox) => {
+          const { project, today, stdout, stderr, releaseSpecificationPath } =
+            await setupFollowMonorepoWorkflow({
+              sandbox,
+              doesReleaseSpecFileExist: true,
             });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: false,
+            today,
+            stdout,
+            stderr,
           });
 
-          describe('when the editor command does not complete successfully', () => {
-            it('removes the release spec file', async () => {
-              await withSandbox(async (sandbox) => {
-                const project = buildMockMonorepoProject();
-                const stdout = fs.createWriteStream('/dev/null');
-                const stderr = fs.createWriteStream('/dev/null');
-                mockDependencies({
-                  determineEditor: {
-                    path: '/some/editor',
-                    args: [],
-                  },
-                });
-                jest
-                  .spyOn(
-                    releaseSpecificationModule,
-                    'waitForUserToEditReleaseSpecification',
-                  )
-                  .mockRejectedValue(new Error('oops'));
-
-                await expect(
-                  followMonorepoWorkflow({
-                    project,
-                    tempDirectoryPath: sandbox.directoryPath,
-                    firstRemovingExistingReleaseSpecification: false,
-                    stdout,
-                    stderr,
-                  }),
-                ).rejects.toThrow(expect.anything());
-
-                await expect(
-                  fs.promises.readFile(
-                    path.join(sandbox.directoryPath, 'RELEASE_SPEC'),
-                    'utf8',
-                  ),
-                ).rejects.toThrow(/^ENOENT: no such file or directory/u);
-              });
+          expect(await fileExists(releaseSpecificationPath)).toBe(false);
+        });
+      });
+
+      it('does not remove the release spec file', async () => {
```

Comment:
Hmm, this looks out of place. This describe block is for when the release spec file exists, but it doesn't exist here

Reply:
Oops! Looks like copypasta. Removed in [9c04d44](https://github.com/MetaMask/create-release-branch/pull/21/commits/9c04d444e2042c16970d3d4dea601af2968b72e7).